src/ucs/type/cpu_set.h

```diff
@@ -0,0 +1,73 @@
+/**
+* Copyright (C) Mellanox Technologies Ltd. 2001-2015.  ALL RIGHTS RESERVED.
+* Copyright (C) The University of Tennessee and The University 
+*               of Tennessee Research Foundation. 2015. ALL RIGHTS RESERVED.
+*
+* See file LICENSE for terms.
+*/
+
+#ifndef UCS_TYPE_CPU_SET_H
+#define UCS_TYPE_CPU_SET_H
+
+#include <stdint.h>
+#include <stddef.h>
+
+/* Type for array elements in 'cpu_set_t'.  */
+typedef unsigned long int ucs_cpu_mask;
```

Comment:
pls fix code style in this file, remove unused macros, and re-write anything that was copied from GPL headers